auv_setup/config/robots/beluga.yaml

```diff
@@ -34,17 +34,19 @@ propulsion:
   thrusters:
     num: 8
     configuration_matrix:
-      [[ -0.70711,  0.00000,  0.00000,  -0.70711,  -0.70711,  0.00000,  0.00000, -0.70711], # Surge
-       [ 0.70711,  0.00000,  0.00000,  -0.70711, 0.70711,  0.00000,  0.00000,-0.70711], # Sway
+
+      [[ 0.70711,  0.00000,  0.00000,  0.70711,  0.70711,  0.00000,  0.00000, 0.70711], # Surge
+       [0.70711,  0.00000,  0.00000,  -0.70711, 0.70711,  0.00000,  0.00000, -0.70711], # Sway
        [ 0.00000,  1.00000,  1.00000,  0.00000,  0.00000,  1.00000,  1.00000, 0.00000], # Heave
-       [ 0.00000, -0.22000, -0.22000,  0.00000,  0.00000,  0.22000,  0.22000, 0.00000], # Roll
-       [ 0.06500, -0.24000,  0.24000,  0.06500,  0.06500,  0.24000, -0.24000, 0.06500], # Pitch 
-       [ 0.32557,  0.00000,  0.00000,  0.32557,  -0.32557, 0.00000,  0.00000,-0.32557]] # Yaw
+
+       [ 0.00000, 0.20600, -0.20600,  0.00000,  0.00000, -0.20600,  0.20600,  0.00000 ], # Roll
+       [ 0.00000, 0.09600,  0.09600,  0.00000,  0.00000, -0.09600, -0.09600,  0.00000 ], # Pitch 
+       [-0.30000, 0.00000,  0.00000,  0.30000,  0.30000,  0.00000,  0.00000, -0.30000 ]] # Yaw
 
     rate_of_change:
       max: 1 # Maximum rate of change in newton per second for a thruster
-    map: [0, 1, 2, 3, 4, 5, 6, 7]  
-    direction: [1, -1, 1, 1, -1, -1, 1, -1]
+    map:  [4, 5, 2, 3, 0, 1, 6, 7] # [0, 1, 2, 3, 4, 5, 6, 7]  
+    direction: [-1, 1, -1, -1, 1, 1, 1, -1]
```

Comment:
Map should be the same between Mk2 v1 and v2, but the direction might change, depending if electro swap some of the phases